README.md

```diff
@@ -61,7 +62,7 @@ npm i domain-functions zod
 import { makeDomainFunction, inputFromForm } from 'domain-functions'
 import * as z from 'zod'
 
-const schema = z.object({ number: z.preprocess(Number, z.number()) })
+const schema = z.object({ number: z.coerce.number() })
```

Comment:
Zod now supports [primitive coercion](https://zod.dev/?id=preprocess) without the need for .preprocess()